test/schema/partner_show.js

```diff
@@ -16,11 +17,17 @@ describe('PartnerShow type', () => {
       displayable: true,
     }));
 
+    total
+      .onCall(0).returns(Promise.resolve(42))
+      .onCall(1).returns(Promise.resolve(2));
+
     PartnerShow.__Rewire__('gravity', gravity);
+    PartnerShow.__Rewire__('total', total);
```

Comment:
The stubbing of `total` works, but when I call it (in `schema/partner_show.js`) it doesn’t appear to return the promises I expect it to and the tests fail with `null` values.

Any idea what I’m doing wrong?